Description: Link transicc and testbed binaries against libmath
 The bug fix for Debian bug #745748 (introduction of ppc64el) requires running
 libtool at build time. This results in some unresolved sybmols due to a missing
 linkage to libmath.
Author: Thomas Weber <tweber@debian.org>
Bug: <URL to the upstream bug report if any, implies patch has been forwarded, optional>
Bug-Debian: https://bugs.debian.org/745748
Forwarded: <URL|no|not-needed, useless if you have a Bug field, optional>
Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/configure.ac
+++ b/configure.ac
@@ -355,6 +355,15 @@
 TIFFICC_DEPLIBS=`echo $TIFFICC_DEPLIBS | sed -e 's/  */ /g'`
 AC_SUBST(TIFFICC_DEPLIBS)
 
+# Libraries that the transicc program depends on
+TRANSICC_DEPLIBS="$LIB_MATH"
+TRANSICC_DEPLIBS=`echo $TRANSICC_DEPLIBS | sed -e 's/  */ /g'`
+AC_SUBST(TRANSICC_DEPLIBS)
+
+# Libraries that the testbed programs depends on
+TESTBED_DEPLIBS="$LIB_MATH"
+TESTBED_DEPLIBS=`echo $TESTBED_DEPLIBS | sed -e 's/  */ /g'`
+AC_SUBST(TESTBED_DEPLIBS)
 LIBS=''
 
 #
--- a/testbed/Makefile.am
+++ b/testbed/Makefile.am
@@ -11,7 +11,7 @@
 
 # CFLAGS = --pedantic -Wall -std=c99 -O2
 
-testcms_LDADD = $(top_builddir)/src/liblcms2.la 
+testcms_LDADD = $(top_builddir)/src/liblcms2.la @TESTBED_DEPLIBS@
 testcms_LDFLAGS = @LDFLAGS@
 testcms_SOURCES = testcms2.c testplugin.c zoo_icc.c testcms2.h
 
--- a/utils/transicc/Makefile.am
+++ b/utils/transicc/Makefile.am
@@ -11,7 +11,7 @@
 
 bin_PROGRAMS = transicc 
 
-transicc_LDADD = $(top_builddir)/src/liblcms2.la 
+transicc_LDADD = $(top_builddir)/src/liblcms2.la  @TRANSICC_DEPLIBS@
 transicc_LDFLAGS = @LDFLAGS@
 transicc_SOURCES = transicc.c ../common/xgetopt.c ../common/vprf.c  ../common/utils.h
 transicc_MANS = transicc.1
